### ISSUE TITLE
Aqara wireless switch D1

### ISSUE THREAD

**Issue by giuliandenicola1** [dresden-elektronik/deconz-rest-plugin#3206]
## Device

- Product name: Aqara Wireless switch D1
- Manufacturer: Xiaomi
- Model identifier: WXKG07LM
- Device type: 
  - Switch

## Screenshots

![Screenshot_20200828-233045_Banggood](https://user-images.githubusercontent.com/67443916/91616693-82f1de00-e986-11ea-981e-769f4b5537f0.jpg)

**Comment by Smanar** [comment]
Hello can you give the capture from deconz ?
Because there is so much model that look like this one and with the same name.

Else it seem this device is already working > https://github.com/dresden-elektronik/deconz-rest-plugin/issues/3066

**Comment by giuliandenicola1** [comment]
> Hello can you give the capture from deconz ?
You mean this?

![Screenshot](https://i.imgur.com/GCMR3Og.png)
![Screenshot](https://i.imgur.com/6PbD2Eh.png)

**Comment by Smanar** [comment]
Yep, but for phoscon we can do nothing with the API, need to ask that to Phoscon dev.

**Comment by Mimiix** [comment]
@giuliandenicola1 I am giving you the benefit of the doubt here and move this to the Phoscon App repo.

In the future: Please make sure to fill the complete template. 
 

@Smanar If you encounter issues like this in the future (Meant for phoscon repo) , feel free to tag me in the issue so i can move them! Thanks ❤️
